### DISCUSSION TITLE
聖誕禮物

### DISCUSSION THREAD

**Discussion by gigitilto** [1101-GS4719-C/Python-Community-GS4719-C#3680] [General]
我的性格孤僻，之前都沒有朋友，但上了大學後，我第一次交到了可以講話的朋友。今天是聖誕節，我想送他聖誕節禮物，以感謝他願意陪我及聽我說話，但我卻對他並不了解，我覺得請他吃飯是一個選擇，但他本人正在飲食控制，我不確定這是否是好的選擇，若我要贈送禮物，我該給他什麼會比較適當？

**Comment by kzx-creat** [comment]
我覺得要看是男生女生還有收禮者的生活習慣耶，但你說得請吃飯感覺不錯!可以事先跟堆方討論想吃什麼，也不會有送禮但送到對方不喜歡的這種問題。

**Comment by qiu0109** [comment]
我覺得可以寫一張卡片再加一樣小東西例如巧克力之類的。冒然送太貴的禮物會讓人感到不自在，所以我覺得卡片是一個很棒的選擇，可以上網查一些手工卡片的教學照著做，表達自己的感謝也傳達出自己的用心。

**Comment by BOSENCHANG2004** [comment]
我認為陪伴就是最好的禮物，送什麼禮物不是重點，如果真的有心陪伴就是給對方最好的禮物。

**Comment by lichilun** [comment]
我覺得，想送禮物感謝朋友是一件很棒的事！但因為他在飲食控制，請吃飯可能不太適合。送一些實用的小禮物，比如保溫瓶、筆記本、香氛小物或圍巾，這些不會太有負擔，也能表達心意。再搭配一張卡片，寫下你的感謝，這樣就很溫暖了。

**Comment by yurukung** [comment]
如果對象是女生的話，可以送精緻的小餅乾，因為量不是太大，所以不會造成別人的困擾，小餅乾的食用期限也比較長，加上女生聖誕節都會互送有些小糖果餅乾，也不會看起來太突兀。如果是男生的話我其實不知道。

**Comment by Welly940504** [comment]
如果是我遇到這種情況，我會想送點不太浮誇、但可以表達我心意的小東西。因為對方正在飲食控制，請吃飯可能會讓他不好意思拒絕，反而壓力大，所以我會避開食物類。

我可能會寫一張卡片，簡單講一下「謝謝你這段時間陪我聊天，真的很開心可以認識你」，然後搭配一個實用的小東西，比如一支好寫的筆、一個好看的書籤，或是那種療癒系的小東西，像香氛片、桌上小公仔什麼的，感覺比較不會出錯，也不會讓對方覺得太負擔。

重點其實不是送什麼，而是你有這份心。他收到的時候一定會覺得很溫暖。
